File: model/demandeJeton.js
var mongoose = require('mongoose');  

let Schema = mongoose.Schema;

var DemandeJetonSchema = Schema({  
  iduser: String,
  emailuser: String,
  jetonsdemande: Number,
  statut: String
});
mongoose.model('demandeJeton', DemandeJetonSchema);

module.exports = mongoose.model('demandeJeton',DemandeJetonSchema);

File: winform/winform-TpTransversal/Model/DemandeJeton.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace winform_TpTransversal.Model
{
    public class DemandeJeton
    {
        public string iduser { get; set; }
        public int jetonsdemande { get; set; }
        public string statut { get; set; }

        public DemandeJeton(string iduser, int jetonsdemande, string statut)
        {
            this.iduser = iduser;
            this.jetonsdemande = jetonsdemande;
            this.statut = statut;
        }
        public DemandeJeton()
        {
        }
    }
}

